### DISCUSSION TITLE
Piggy Banks: add an option for a piggy without a goal (target amount)

### DISCUSSION THREAD

**Discussion by macuros** [firefly-iii/firefly-iii#5984] [Firefly III] [answered]
Regarding this: https://github.com/firefly-iii/firefly-iii/issues/5269

Copy/paste my last answer from there: 

Hmm, thanks for adding that, but I'm not sure how to set a piggy bank without the target date. I would like to set aside fixed amount every month, or even add a bit more.

UPDATE:
Actually, will the target amount reset every month? I can set it without a target date, but target amount needs to be filled in. I set it to 50 units, add it, and now I can't add any more units to it.

**Comment by JC5** [comment]
You can leave the target date empty.

**Comment by macuros** [comment]
Yes, but what about this part:

Actually, will the target amount reset every month? I can set it without a target date, but the target amount needs to be filled in. I set it to 50 units, add it, and now I can't add any more units to it.

**Reply by JC5** [reply]
No the target amount will not reset. Once you filled the piggy bank it's full.

**Comment by macuros** [comment]
Oh, hm, it would be nice to have one without the target amount. Just to add the same amount each month. Or even add a bit more if you wish.

**Reply by JC5** [reply]
Yes you can do that of course. But Firefly III won't auto suggest the amount.

If you want that, set the goal you had in mind and a target date, and Firefly III will calculate what you need to save each month. If you set no date and no target, Firefly III won't suggest anything. If you want a reminder, put it in the title of the piggy bank.

**Comment by macuros** [comment]
Understand, but I can't create a piggy bank without a target amount. It just throws an error.

**Comment by JC5** [comment, marked as answer]
Mmmm, try amount "0". It could be a bug, I'll make sure irs fixed in the next release!

**Comment by macuros** [comment]
Yes, that's what I was missing. Now it's all clear. Thank you!
